src/graphql/products/createVariant.ts

```diff
@@ -0,0 +1,23 @@
+export default `
+mutation productVariantsBulkCreate($productId: ID!, $variants: [ProductVariantsBulkInput!]!, $productPublishInput: ProductPublishInput!) {
```

Comment:
```suggestion
mutation ($productId: ID!, $variants: [ProductVariantsBulkInput!]!, $productPublishInput: ProductPublishInput!) {
```

Have been doing this for readability lol